gems/smithy-client/sig/smithy-client/errors.rbs

```diff
@@ -2,7 +2,7 @@ module Smithy
   module Client
     module Errors
       class ServiceError < RuntimeError
-        def initialize: (HandlerContext, String?, untyped?) -> void
+        def initialize: (HandlerContext?, String?, untyped?) -> void
```

Comment:
May want to update the documentation for this - not clear that these are supposed to be optional,